examples/ssr-caching/server.js

```diff
@@ -9,7 +9,10 @@ const handle = app.getRequestHandler()
 
 // This is where we cache our rendered HTML pages
 const ssrCache = new LRUCache({
-  max: 100,
+  length: function (n, key) {
+    return n.toString().length + key.toString().length
+  },
+  max: 100 * 1000 * 1000, // 100MB cache soft limit
```

Comment:
Shouldn't this be `1024` instead of `1000`, not that it really matter 😄

Reply:
@timneutkens  Yeah, should be. I'll fix later. for some reason I'm not seeing it in canary yet. I've had updating this example on my to-do list for  a while and just needed an accomplishment of any kind today.